yt_dlp/extractor/prankcast.py

```diff
@@ -0,0 +1,79 @@
+import json
+from datetime import datetime
+from .common import InfoExtractor
+from ..utils import parse_duration
+
+
+class PrankCastIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?prankcast\.com/.*/showreel/(?P<id>[0-9]+)'
+    _TESTS = [{
+        'url': 'https://prankcast.com/Devonanustart/showreel/1561-Beverly-is-back-like-a-heart-attack-',
+        'info_dict': {
+            'id': '1561',
+            'ext': 'mp3',
+            'title': 'Beverly is back like a heart attack!',
+            'uploader': 'Devonanustart',
+            'upload_date': '20220825'
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        # Extract the JSON
+        json_info = self._search_nextjs_data(webpage, video_id)
+
+        # Get the broadcast URL and the recording hash.
+        # The full URL is {broadcast_url}/{recording_hash}.mp3
+        broadcast_url = json_info['props']['pageProps']['ssr_data_showreel']['broadcast_url']
+        recording_hash = json_info['props']['pageProps']['ssr_data_showreel']['recording_hash']
+        url = broadcast_url + recording_hash + ".mp3"
+
+        # Get dates
+        start_date = None
+        if 'start_date' in json_info['props']['pageProps']['ssr_data_showreel']:
```

Comment:
This block of code can rewritten something like:
``` var = json_info.get('key')```

Reply:
This line (line 34-35) shouldn't fatal, right ?. Use `traverse_obj` from `utils`

Reply:
Is using ".get(key)" like I did in the last commit okay? "traverse_obj" seems redundant since I'm already getting the "['props']['pageProps']['ssr_data_showreel']" path when parsing the "json_info" (using the changes you proposed above). If "traverse_obj" is still required let me know and I'll change it, the contribution guide says both are OK but I'm not sure when to use one or the other...